packages/webdriver/src/request/index.ts

```diff
@@ -203,6 +203,7 @@ export default abstract class WebDriverRequest extends EventEmitter {
              * handle timeouts
              */
             if ((response as RequestLibError).code === 'ETIMEDOUT') {
+                extendTimeoutErrByExecCmd(response, fullRequestOptions)
                 return retry(response, 'Request timed out! Consider increasing the "connectionRetryTimeout" option.')
```

Comment:
I suggest instead of modifying the response error object, to return a new error we can continue to use. Maybe let's rename `extendTimeoutErrByExecCmd` to `getTimeoutError`
```suggestion
                const error = extendTimeoutErrByExecCmd(response, fullRequestOptions)
                return retry(error, 'Request timed out! Consider increasing the "connectionRetryTimeout" option.')
```